COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
Listing Google Inc. as the only explicitly mentioned copyright holder next to Eelco does feel excessive. I also doubt that it is necessary to do so. Generally speaking, we need a better policy with regard to (c) issues that what we have today, i.e. mentioned Eelco as the only (c) holder doesn't feel quite right either. Anyway, he's certainly the most important contributor to Nixpkgs, so the status quo feels okay'ish to me until we have figured out a better solution.